docusaurus/docs/e2e-test-a-plugin/ci.md

```diff
@@ -0,0 +1,86 @@
+:::
+
+<details>
+  <summary> <h3>Backend plugin workflow</h3> </summary>
+  <CodeSnippets
+snippets={[
+{ component: BEPluginWorkflowNPM, label: 'npm' },
+{ component: BEPluginWorkflowYarn, label: 'yarn' },
+{ component: BEPluginWorkflowPNPM, label: 'pnpm' }
+]}
+groupId="package-manager"
+queryString="current-package-manager"
+/>
+</details>
+
+<details>
+  <summary> <h3>Frontend plugin workflow</h3> </summary>
+  <CodeSnippets
+snippets={[
+{ component: BEPluginWorkflowNPM, label: 'npm' },
+{ component: BEPluginWorkflowYarn, label: 'yarn' },
+{ component: BEPluginWorkflowPNPM, label: 'pnpm' }
+]}
+groupId="package-manager"
+queryString="current-package-manager"
+/>
+</details>
+
+### The e2e-versions Action
+
+The example workflows have a job called `Resolve Grafana images` that uses the [e2e-version](https://github.com/grafana/plugin-actions/tree/main/e2e-version) Action to resolve a list of Grafana images. For every image returned, a new job will be fired up that builds the plugin, starts Grafana and runs the end-to-end tests.
+
+The Action supports two modes - `plugin-grafana-dependency` and `version-support-policy`.
+
+#### plugin-grafana-dependency mode
+
+By default, the e2e-version Action will return the most recent grafana-dev image plus all the latest patch releases of every minor version of Grafana Enterprise since the version that was specified as `grafanaDependency` in the [plugin.json](../metadata.md). To avoid starting too many jobs, to output will be capped 6 versions.
+
+![](/img/e2e-version-plugin-dependency.png)
+
+#### version-support-policy mode
+
+Except for resolving the most recent grafana-dev image, the `version-support-policy` mode will resolve versions according to Grafana's plugin compatibility support policy. Specifically, it retrieves the latest patch release for each minor version within the current major version of Grafana. Additionally, it includes the most recent release for the latest minor version of the previous major Grafana version.
+
+![](/img/e2e-version-version-support-policy.png)
+
+To use the `version-support-policy` mode you need to specify the `version-resolver-type` input argument.
+
+```yml
+- name: Resolve Grafana E2E versions
+        id: resolve-versions
+        uses: grafana/plugin-actions/e2e-version
+        with:
+          version-resolver-type: version-support-policy
+```
+
+### Playwright report
+
+A Playwright HTML test report is generated for every Grafana version that is being tested. In case any of the tests failed, a Playwright trace viewer is also generated along with the report. The `Upload artifacts` step in the example workflows uploads the report as artifacts. To find information on how to download and view the report, refer to the Playwright [documentatation](https://playwright.dev/docs/ci-intro#html-report).
```

Comment:
```suggestion
A Playwright HTML test report is generated for every Grafana version that is being tested. In case any of the tests fail, a Playwright trace viewer is also generated along with the report. The `Upload artifacts` step in the example workflows uploads the report as an artifact. To find information on how to download and view the report, refer to the [Playwright documentatation](https://playwright.dev/docs/ci-intro#html-report).
```